build-logic/src/main/kotlin/polaris-root.gradle.kts

```diff
@@ -57,3 +61,12 @@ if (System.getProperty("idea.sync.active").toBoolean()) {
     }
   }
 }
+
+extensions.getByType<PublishingHelperExtension>().apply {
+  asfProjectName = "polaris"
+
+  mailingLists.addAll("dev", "issues", "commits")
+
+  podlingPpmcAsfIds.addAll("anoop", "ashvin", "jackye", "russellspitzer", "snazy", "vvcephei")
```

Comment:
I would add the mentors here (very selfish, but I would like to be on the list :) ).

Reply:
It was a trick to verify that you review this change ;) ;) :disappear: